### PR TITLE
Condarise: remove old packages

### PR DESCRIPTION
**Pull request by vladsaveliev**

### CODE REVIEW

File: .github/workflows/condarise.yaml [hunk 1]
```diff
@@ -37,6 +37,12 @@ jobs:
       - name: Build package
         run: conda build conda/hail
 
+      - name: Remove old versions
+        run: |
+          anaconda -t ${{ secrets.ANACONDA_TOKEN }} show cpg/hail 2>&1 \
+          | grep "^   +" | head -n-8 | sed 's#   \+ ##' \
```
Comment (lgruen) [inline .github/workflows/condarise.yaml:43]: `head -n-8` -- did you mean `-n 8`?

Comment (vladsaveliev) [reply .github/workflows/condarise.yaml:43]: No, I meant `head -n-8`, which is "return all but last 8 lines". The latest packages go last.

Comment (lgruen) [reply .github/workflows/condarise.yaml:43]: TIL!
(On macOS, this returns `head: illegal line count -- -8`.)

Comment (vladsaveliev) [reply .github/workflows/condarise.yaml]: Oh, good catch. I'm using gnu head on my laptop.

Wonder how it will go with the CI macos though.

Comment (vladsaveliev) [reply .github/workflows/condarise.yaml]: Ah, we need to run the clean-up step once anyway, so will do that for ubuntu only.

**Review by lgruen — APPROVED** [review]
What a nice magic incantation! :)

**Pull request merged**